### PR TITLE
Add accessibility tests for organization enrollment steps

### PR DESCRIPTION
**Pull request by PaulMorris**
These accessibility tests cover the main pages of the enrollment process for organizations.  Includes an easy fix for one failing test (a color contrast issue).

Tested by running the tests.  The two that aren't `@ignore`d pass successfully.

Issue #518 Implement accessibility checking in CI

### CODE REVIEW

File: psm-app/integration-tests/src/test/resources/features/enrollment/accessibility-organization.feature [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  # fails with duplicate IDs and inputs without labels/titles
+  @ignore
+  Scenario: 2. Individual Member Info Page
+    Given I have started an enrollment
+    When I move to the individual member info page
+    When I open an individual member panel
+    Then I should have no accessibility issues
+
+  # fails with duplicate IDs and inputs without labels/titles
```
Comment (cecilia-donnelly) [inline psm-app/integration-tests/src/test/resources/features/enrollment/accessibility-organization.feature]: We still have missing labels/titles?

File: psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/default/ownership_information.jsp [hunk 2]
```diff
@@ -331,7 +331,7 @@
 </div>
 
 <div class="tableBottom">
-    <a href="javascript:;" id="addOwnership">+ Add Another Individual Person Ownership or Control Interest</a>
+    <a href="javascript:;" id="addIndividualOwnership">+ Add Another Individual Person Ownership or Control Interest</a>
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/default/ownership_information.jsp]: @PaulMorris, is this to reduce ID duplication?  (I don't see any remaining instances of `addOwnership`, so just curious what the motivation is.)

File: psm-app/cms-web/WebContent/css/style.css [hunk 3]
```diff
@@ -1388,7 +1388,7 @@ input.date{
   width:20%;
   float:left;
   display:inline;
-  color:#999999;
+  color:#737373;
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/css/style.css]: @PaulMorris, I couldn't see what this changed -- help?

File: psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/default/ownership_information.jsp (hunk 2 above)
Comment (PaulMorris) [reply psm-app/cms-web/WebContent/WEB-INF/pages/provider/enrollment/steps/pageTemplates/default/ownership_information.jsp]: In the java code it's always specified as either individualOwnership or businessOwnership, so this was just to bring these IDs in line with being explicit about what kind of ownership.

File: psm-app/integration-tests/src/test/resources/features/enrollment/accessibility-organization.feature (hunk 1 above)
Comment (PaulMorris) [reply psm-app/integration-tests/src/test/resources/features/enrollment/accessibility-organization.feature]: Alas, it seems so in a few places.  (sigh)  The good news is these tests will make sure we get them all fixed and keep them that way.

File: psm-app/cms-web/WebContent/css/style.css (hunk 3 above)
Comment (PaulMorris) [reply psm-app/cms-web/WebContent/css/style.css]: @cecilia-donnelly This was darkening the lighter text like the "MM/DD/YYYY" in this screenshot:

screenshot-2018-2-15 member information

Comment (cecilia-donnelly) [reply psm-app/cms-web/WebContent/css/style.css]: Aha! Thank you. :)

**Comment by PaulMorris** [comment]
This version is rebased on the changes from PR #667, adds a test for the "facility credentials" page, and includes some further refactoring.

File: psm-app/integration-tests/src/test/resources/features/enrollment/accessibility-organization.feature [hunk 4]
```diff
@@ -11,6 +11,14 @@ Feature: Organization Enrollment Steps Accessibility Checks
     When I am on the organization page
     Then I should have no accessibility issues
 
+  # fails with inputs without labels/titles
```
Comment (cecilia-donnelly) [inline psm-app/integration-tests/src/test/resources/features/enrollment/accessibility-organization.feature]: Alas.

**Pull request merged**